src/hotspot/share/opto/vectornode.cpp

```diff
@@ -2083,6 +2084,114 @@ Node* VectorBlendNode::Identity(PhaseGVN* phase) {
   return this;
 }
 
+Node* SelectFromTwoVectorNode::Ideal(PhaseGVN* phase, bool can_reshape) {
+  int num_elem = vect_type()->length();
+  BasicType elem_bt = vect_type()->element_basic_type();
+  if (Matcher::match_rule_supported_vector(Op_SelectFromTwoVector, num_elem, elem_bt)) {
+    return nullptr;
+  }
+  Node* index_vec = in(1);
+  Node* src1  = in(2);
+  Node* src2  = in(3);
+
+  // Lower the IR to constituents operations.
+  //   SelectFromTwoVectorNode =
+  //     (VectorBlend
+  //         (VectorRearrange SRC1, INDEX)
+  //         (VectorRearrange SRC2, NORM_INDEX)
```

Comment:
```suggestion
  //         (VectorRearrange SRC1 INDEX)
  //         (VectorRearrange SRC2 NORM_INDEX)
```
Either consistently use commas or none at all ;)